modules/cli/apply/apply.go

```diff
@@ -129,33 +117,20 @@ func readBundle(ctx context.Context, name, baseDir string, opts *Options) (*bund
 		SyncGeneration:  opts.SyncGeneration,
 		Auth:            opts.Auth,
 	})
-}
-
-func createName(name, baseDir string) string {
```

Comment:
It makes sense but the original reason for adding the path is to make bundle name unique when they are creating under gitRepo. Let's say you will have following git repository:
```
dir/foo/bar/fleet.yaml
dir/foo1/bar/fleet.yaml
```
If I pass dir to fleet CLI `fleet apply foo ./dir` and let it create bundles without considering path, It will try to create two bundle with the same name `foo`, causing conflicts. I understand that this is confusing as it should really be BUNDLE_NAME_PREFIX instead of BUNDLE_NAME as we always append path to avoid duplication. Changing this behavior will likely to break existing functionlities. If you like the ability to specify the name we are already adding that in this PR https://github.com/rancher/fleet/pull/552.

Reply:
Gotcha, makes sense. I pushed the adding of that part back.